### PR TITLE
Recalibrate Aristarchus multi-agent code-review pipeline

### PR DESCRIPTION
**Pull request by dobesv**
Fixes the broken template-include architecture by removing transitive includes from shared core files and wiring policy injection at the wrapper level. Eliminates the mandatory-coverage-checks "double-jeopardy" policy, expanding coverage exemptions from 7 to 11.

Introduces several review gates and refinements:
- Introduced-by-diff gate to focus on PR changes
- PR-type and blast-radius scaling for Draft/WIP reviews
- Cross-round finding retirement for contested or justified issues
- Anchoring gate to enforce specific code locations in findings
- Visual/render APPROVE guard and missing-metadata fallbacks

Adds Opis, a new Performance & Scalability Muse, and wires the muse-output-format schema into all 11 Muses with verification clauses. Applies calibration refinements across all Muses and adds a docs entry for the dual-repo prompt architecture.

Plan: aristarchus-prompt-improvements

## Summary by CodeRabbit

* **New Features**
  * Added a Muse output-format and verification contract to constrain and validate Muse findings.
  * Added a performance & scalability specialist workflow for identifying production-impacting inefficiencies.
  * Introduced a mandatory automated test coverage policy with defined exemptions and enforcement.

* **Improvements**
  * Strengthened review quality with an evidence “anchoring gate,” calibrated severity examples, and tighter suppression/scope rules across key checks (security, reliability, accessibility, privacy, i18n).
  * Expanded deployment/CI build-config and API/backward-compatibility verification guidance.

* **Documentation**
  * Added a guide to prevent dual-repository template/include rendering mismatches and included an end-to-end verification approach.

### CODE REVIEW

File: docs/solutions/workflow-issues/dual-repo-pantheon-prompt-architecture-2026-07-18.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+## Symptoms
+
+```
+- harnx prompts rendered with literal `{{ include "shared/policy-..." }}` text passed to LLM
+- MiniJinja has NO {{ include }} support — variables only expand in top-level wrapper body
+- k8s includes resolved by configMap KEY (e.g., `policy-test-coverage`) not filesystem path (`policies/test-coverage`)
+- No verification harness existed to detect unrendered directives before production
+- Coverage double-jeopardy: three Judges each scanned for coverage issues independently
+- Calibration drift: ~15 per-Muse refinements needed across both repos in tandem
+```
```
[the remark this answers is not in the document]
Comment (dobesv) [reply docs/solutions/workflow-issues/dual-repo-pantheon-prompt-architecture-2026-07-18.md]: Fixed in `37155b981` — added the `text` language tag to the Symptoms fenced block; contents unchanged.

File: docs/solutions/workflow-issues/dual-repo-pantheon-prompt-architecture-2026-07-18.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+| k8s-manifests | kagent preprocessor | Recursive `{{ include "shared/KEY" }}` at any depth | Resolved transitively |
+
+**k8s configMap key resolution:**
+Invoking `{{ include "shared/policies/test-coverage" }}` fails silently — kagent resolves by configMap KEY registered in `shared/kustomization.yaml`, not filesystem path. Correct form: `{{ include "shared/policy-test-coverage" }}` matching the key `policy-test-coverage`.
+
+**Architecture requirement:**
+Shared core files must contain NO `{{ include }}` and NO cross-file `{{varname}}`. All composition happens at wrapper level:
+- harnx: declare `variables:` entry, reference `{{varname}}` in wrapper body
+- k8s: place `{{ include }}` directly in wrapper or core (kagent resolves recursively)
+
```
[the remark this answers is not in the document]
Comment (dobesv) [reply docs/solutions/workflow-issues/dual-repo-pantheon-prompt-architecture-2026-07-18.md]: Fixed in `37155b981` — split the Architecture requirement into engine-specific rules: harnx (MiniJinja) shared cores avoid includes and cross-file vars with composition declared/referenced in wrappers; k8s (kagent) shared-core includes are explicitly allowed and resolve recursively. Removed the blanket both-engine prohibition; each engine keeps its distinct syntax/behavior.

File: docs/solutions/workflow-issues/dual-repo-pantheon-prompt-architecture-2026-07-18.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+# Capture baseline renders
+for agent in minos rhadamanthus aeacus thalia aristarchus; do
+  HARNX_CONFIG_DIR=$VERIFY_DIR harnx info agent pantheon/$agent > /tmp/baseline-$agent.txt
+done
+
+# Check for leaks
+grep -n '{{ include\|{{[a-z_]\+}}' /tmp/baseline-*.txt
+```
+
+Caveat: `harnx info agent` also dumps variable-default metadata — rendered strings appear ~twice. Judge the prompt BODY, not metadata dump.
```
[the remark this answers is not in the document]
Comment (dobesv) [reply docs/solutions/workflow-issues/dual-repo-pantheon-prompt-architecture-2026-07-18.md]: Fixed in `37155b981` — the verification example now extracts only each agent's rendered prompt body (awk-strips the frontmatter + variable-default metadata dump) before the leak grep, so expected placeholders in metadata no longer false-positive. Metadata caveat retained.

File: packages/pantheon/agents/shared/aristarchus.md [hunk 4]
```diff
@@ -46,6 +44,7 @@ An unresolved Blocker finding for missing test coverage MUST result in a `REQUES
 | Terpsichore | Refactoring & completeness | — |
 | Urania | Architecture & big picture | oracle, pytheas |
 | Nemesis | Reliability & error handling | — |
+| Opis | Performance & scalability | — |
```
[the remark this answers is not in the document]
Comment (dobesv) [reply packages/pantheon/agents/shared/aristarchus.md:47]: Fixed in `37155b981` — added `opis_session_prompt` to the coordinator wrapper's `use_tools` (`packages/pantheon/agents/aristarchus.md`), so Opis is now invocable. The k8s equivalent (Opis in `aristarchus.agent.yaml` A2A delegation) was already wired.

File: packages/pantheon/agents/shared/aristarchus.md [hunk 5]
```diff
@@ -101,6 +106,11 @@ Five phases, tracked in the plan. Each phase is a task. Plans are kept for futur
...
+- **Introduced-by-diff gate**: Before confirming any Blocker, verify it is directly introduced or materially worsened by the changes in this PR. Do not confirm Blockers for pre-existing patterns unless the PR modifies those specific lines in a way that creates a regression. Pre-existing issues in untouched code must be downgraded to Non-blocking issue or Suggestion.
+- **Draft/WIP blast-radius cap**: If `pr_type` is draft, wip, strawman, demo, or one-liner — cap the report to the top 2–3 promotion-blocking Blockers only. Do not raise NEEDS_DISCUSSION or REQUEST_CHANGES for zero-impact latent edge cases. Prepend the report: "Draft/WIP review — only promotion-blocking issues shown."
+- **Cross-round retirement**: For each Blocker, check `prior-round-findings`. If the same finding (same file + same issue class) was posted in a prior round AND either (a) the author responded with a documented trade-off justification, OR (b) Judges failed 2-of-3 consensus on it in a prior round — downgrade to Suggestion and append: "[Retired — contested in prior rounds; see history]". Do not re-assert a contested Blocker without new evidence from the current diff.
```
[the remark this answers is not in the document]
Comment (dobesv) [reply packages/pantheon/agents/shared/aristarchus.md]: Fixed in `37155b981` (k8s parity `7a739f82502`) — the Cross-round retirement rule now (a) explicitly excludes mandatory test-coverage Blockers (they retain Blocker status + REQUEST_CHANGES unless a documented coverage exemption/opt-out applies), and (b) matches prior findings using a precise fingerprint (file path + line/range + issue class + specific symbol/rule cited), not file + issue class alone.

File: packages/pantheon/agents/shared/policy-test-coverage.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+There are no exceptions beyond the documented exemption list and a
+properly formatted opt-out justification.
+
+### Closed Exemption List
+
+The following eleven categories are the only recognized exemptions:
+
+1. **docs-only** — changes confined to documentation files (e.g. `.md`, `.txt`, `docs/`)
+2. **comment-only** — changes that add, remove, or modify inline code comments only
+   - Note: This also falls within the trigger exclusions defined above and requires no opt-out documentation.
```
[the remark this answers is not in the document]
Comment (dobesv) [reply packages/pantheon/agents/shared/policy-test-coverage.md]: Fixed in `37155b981` (k8s parity `7a739f82502`) — the `docs-only` exemption is now restricted to inert documentation paths (docs/, READMEs, changelogs) and explicitly does NOT cover runtime prompt templates/deployed prompt artifacts under `packages/pantheon/agents/` or `configuration/kagent/agents/` — those require rendered-prompt verification.

File: packages/pantheon/agents/shared/polyhymnia.md [hunk 7]
```diff
@@ -95,6 +95,11 @@ and meet regulatory requirements like GDPR, CCPA, and other data protection laws
 - Verify that users can opt-out of tracking.
 - Evaluate transparency about tracking practices.
 
+## Suppression Rules
+- Opaque campaign IDs, click IDs, session tokens, and plain resource IDs (section, org, item, content) do not qualify as PII — they do not identify a person without linkage to additional data. Do not flag these as privacy violations.
+- Baseline-comparison privacy claims (e.g. "this is worse than the existing approach") must be framed as Questions, not findings, unless the diff itself introduces the exposure.
+- Confirm the diff introduces the data handling before raising a Blocker. Pre-existing patterns in unchanged code are advisory notes.
```
[the remark this answers is not in the document]
Comment (dobesv) [reply packages/pantheon/agents/shared/polyhymnia.md:101]: Fixed in `37155b981` (k8s parity `7a739f82502`) — removed session tokens from the blanket non-PII exemption; session/auth tokens (and any linkable/impersonation-capable identifier) are kept in scope for privacy and security review. Campaign/click/resource IDs narrowed to those demonstrably non-linkable and non-sensitive. Baseline-comparison and pre-existing-data rules unchanged.

File: packages/pantheon/agents/shared/thalia.md [hunk 8]
```diff
@@ -10,14 +10,19 @@ and prevent regressions.
 - Look for gaps between what the code does and what the tests verify.
 - Consider the intent of the code, not just its happy path.
 - Identify patterns of under-testing and over-testing.
+- Beyond verifying that a test file exists, verify test substance:
+  - Does the test target the specific changed branch or logic path — would it fail if that branch were deleted or inverted?
+  - Does it assert both the success case and at least one failure/edge case for the changed logic?
+  - Is it actually enabled in CI — not skipped, not in a pending/xit/xdescribe block, not excluded from the CI test run configuration?
+  - Can you connect the changed logic to the specific test file or test case that exercises it? If not, flag the gap.
+
+  Do not Highlight or praise test coverage without confirming substance. A test that always passes regardless of the implementation is not coverage.
```
[the remark this answers is not in the document]
Comment (dobesv) [reply packages/pantheon/agents/shared/thalia.md:19]: Fixed in `37155b981` (k8s parity `7a739f82502`) — failure/edge-case coverage is now conditional: required only when the changed logic has meaningful failure behavior or applicable boundaries. Success-case coverage plus the targeted-test and CI-enabled checks are retained.
